examples/raylib/build.zig

```diff
@@ -0,0 +1,86 @@
+const android = @import("android");
+const std = @import("std");
+
+//this is targeting android API level 29. 
+//You may have to change the values here and in android/AndroidManifest.xml to target your desired API level
+const android_api = "29";
+const android_version = .android10;
+
+pub fn build(b: *std.Build) void {
+    const root_target = b.standardTargetOptions(.{});
+    const optimize = b.standardOptimizeOption(.{});
+    const android_targets = android.standardTargets(b, root_target);
+
+    var root_target_single = [_]std.Build.ResolvedTarget{root_target};
+    const targets: []std.Build.ResolvedTarget = if (android_targets.len == 0)
+        root_target_single[0..]
+    else
+        android_targets;
+
+    const android_apk: ?*android.APK = blk: {
+        if (android_targets.len == 0) {
+            break :blk null;
+        }
+        const android_tools = android.Tools.create(b, .{
+            .api_level = android_version,
+            .build_tools_version = "35.0.0",
+            .ndk_version = "27.2.12479018",
```

Comment:
If possible, update to match other examples. 
```zig
.build_tools_version = "35.0.1",
.ndk_version = "29.0.13113456",
```

As per SDL2 example, it was updated because:
```zig
// NOTE(jae): 2025-03-09
// Previously this example used 'ndk' "27.0.12077973".
//
// However that has issues with the latest SDL2 version when including 'hardware_buffer.h'
// for 32-bit builds.
//
// - AHARDWAREBUFFER_USAGE_FRONT_BUFFER = 1UL << 32
//  - ndk/27.0.12077973/toolchains/llvm/prebuilt/{OS}-x86_64/sysroot/usr/include/android/hardware_buffer.h:322:42:
//  - error: expression is not an integral constant expression
```